pkg/urbit/noun/jets.c

```diff
@@ -1803,6 +1803,7 @@ _cj_print_tas(FILE* fh, u3_noun tas)
   u3r_bytes(0, met_w, (c3_y*)str_c, tas);
   str_c[met_w] = 0;
   fprintf(fh, "/%s", str_c);
+  u3l_log("/%s", str_c);
```

Comment:
Looking at where this gets called, this is gonna go to _both_ strerr, and wherever 
`u3l_log` outputs to (also stderr?). Are you intentionally printing this twice?

Reply:
Good catch! The write to stderr is clobbered by the `u3l_log()` calls, but I didn't mean to leave it.